deeplake/util/connect_dataset.py

```diff
@@ -0,0 +1,68 @@
+from typing import Tuple, Optional
+from deeplake.client.client import DeepLakeBackendClient
+from deeplake.client.log import logger
+from deeplake.util.path import is_hub_cloud_path, get_path_type, get_org_id_and_ds_name
+from deeplake.util.exceptions import InvalidDestinationPathError, InvalidSourcePathError
+from deeplake.util.logging import log_visualizer_link
+
+
+def log_dataset_connection_success(ds_path: str):
+    logger.info("Dataset connected successfully.")
+    log_visualizer_link(ds_path)
+
+
```

Comment:
I think in connect_dataset_entrywe have too much code for one method.

It would be better to separate the code. For instance, we can create a class like this:


```suggestion
class DatasetEntry:
    def __init__(self, token, src_path, dest_path, org_id, ds_name):
        self.token = token
        self.src_path = src_path
        self.dest_path = dest_path
        self.org_id = org_id
        self.ds_name = ds_name
        self.client = DeepLakeBackendClient(token)
        self.org_id, self.ds_name = _get_org_id_and_ds_name(
             dest_path=dest_path, org_id=org_id, ds_name=ds_name
        )
  
    def check_errors(self):
        if is_hub_cloud_path(src_path):
            raise InvalidSourcePathError(
                "Source dataset is already accessible by a Deep Lake path."
         )
         
         if not is_path_connectable(src_path):
             raise InvalidSourcePathError(
                 f"Source path may only be an s3 or gcs path. Got {src_path}."
         )
     
     def connect_dataset_entry(self):
         connected_id = client.connect_dataset_entry(
             src_path=self.src_path, 
             org_id=self.org_id, 
             ds_name=self.ds_name, 
             creds_key=self.creds_key,
     )
```

Reply:
Agreed. I have tried to move the code to a dedicated class, which seems better. Just a little concerned about the names.

Reply:
Agree about names, you have more context about these classes, so most probably you can give better naming